app/views/record/_extended_info.html.erb

```diff
@@ -4,91 +4,83 @@
   <% search_prefix = ENV['EDS_PROFILE_URI'] %>
 <% end %>
 
-<h3 class="subtitle3">More information</h3>
-
-<ul>
+<h3 class="section-title">More information</h3>
+<ul class="list-moreinfo">
   <% if @record.eds_publication_type.present? %>
     <li>
-      Document type: <%= @record.eds_publication_type %>
-    </li>
+      <span class="label">Document type:</span> <%= @record.eds_publication_type %>
+    </li> 
   <% end %>
 
   <% if @record.eds_source_title.present? %>
     <li>
-      Source: <%= @record.eds_source_title %>
-    </li>
+      <span class="label">Source:</span> <%= @record.eds_source_title %>
+    </li> 
   <% end %>
 
   <% if @record.eds_authors.present? %>
     <li>
-      <ul>
-        <% @record.eds_authors.each do |author| %>
-          <li>
-            <%= author %>
-          </li>
-        <% end %>
-      </ul>
-    </li>
+      <span class="label">Authors:</span> 
+      <% @record.eds_authors.each do |author| %>
+        <span class="record-author"><%= author %></span>
+      <% end %>
+    </li> 
   <% end %>
 
   <% if clean_affiliations.present? %>
     <li>
-      <ul>
-        <% clean_affiliations.each do |aff| %>
-          <li>
-            <%= aff %>
-          </li>
-        <% end %>
-      </ul>
-    </li>
+      <span class="label">Affiliations:</span> 
+      <% clean_affiliations.each do |aff| %>
+        <%= aff %>; 
```

Comment:
This will have a semicolon after every affiliation, including the last one in the list; do you want to omit the semicolon after the last one?

Also "Affiliations" will be plural even if there is only one affiliation - do you want us to be clever about that?

Reply:
I like those clever ideas. ++